dashboard/modules/job/tests/test_backwards_compatibility.py

```diff
@@ -76,19 +76,20 @@ def test_error_message():
     )
     wait_for_condition(lambda: client.get_job_status(job_id) == JobStatus.SUCCEEDED)
 
-    # `entrypoint_num_cpus`, `entrypoint_num_gpus`, and `entrypoint_resources`
-    # are not supported in ray<2.2.0.
+    # `entrypoint_num_cpus`, `entrypoint_num_gpus`, `entrypoint_memory`, and
+    # `entrypoint_resources` are not supported in ray<2.2.0.
```

Comment:
Please update this comment to reflect that `entrypoint_memory` is not supported in ray<2.8.0